config/m4/compiler.m4

```diff
@@ -165,6 +166,60 @@ AS_IF([test "x$with_avx" != xyes],
       ])
 
 
+AC_ARG_ENABLE(aarch64-tuned,
+	AC_HELP_STRING([--enable-aarch64-tuned], [Enable ARM 64bit micro-architecture specific optimizations, default: NO]),
```

Comment:
maybe do it as part of --enable-optimizations?
(it's turned off by default for RPM, but otherwise it's on)

Reply:
I actually was really confused by the flag :) Seems like it has to be used in combination with avx/sse flags, otherwise the vector optimizations are ignored. These combination of having 2 flags for a things to work is super odd. 

Regardless, I don't mind integrating this under enable optimizations. Technically you can extend the logic to Power and x86.